lib/internal/process/pre_execution.js

```diff
@@ -578,11 +578,18 @@ function initializeESMLoader() {
 }
 
 function initializeSourceMapsHandlers() {
-  const { setSourceMapsEnabled, getSourceMapsEnabled } =
+  const { initializeSourceMapsEnabled, setSourceMapsEnabled, getSourceMapsEnabled } =
```

Comment:
We don't need `initializeSourceMapsEnabled`. Since in this scope, we already have `setSourceMapsEnabled` and `getOptionValue`.

Reply:
That makes sense.
Updated: https://github.com/nodejs/node/pull/46391/commits/f16d225f2d734c8eff463a48de391cbeaf37328f